hivesim-ts/simulators/ts-sim-demo/main.ts

```diff
@@ -0,0 +1,162 @@
+import exp from "constants";
+import { Simulation } from "../../simulation.js";
```

Comment:
This file does 2 space indenting others use 4 spaces we should stick with 1 standard for consistency. This is more of a overall remark as there are a few files which use both.